dev/com.ibm.websphere.appserver.features/visibility/public/jsp-3.0/com.ibm.websphere.appserver.jsp-3.0.feature

```diff
@@ -0,0 +1,48 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jsp-3.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: jakarta.servlet.jsp;  type="spec", \
+ jakarta.servlet.jsp.el;  type="spec", \
+ jakarta.servlet.jsp.tagext;  type="spec", \
+ jakarta.servlet.jsp.jstl.sql;  type="spec", \
+ jakarta.servlet.jsp.jstl.tlv;  type="spec", \
+ jakarta.servlet.jsp.jstl.core;  type="spec", \
+ jakarta.servlet.jsp.jstl.fmt;  type="spec", \
+ jakarta.el;  type="spec", \
+ org.apache.taglibs.standard;  type="internal", \
+ org.apache.taglibs.standard.tag.common.core;  type="internal", \
+ org.apache.taglibs.standard.tag.common.fmt;  type="internal", \
+ org.apache.taglibs.standard.tag.common.sql;  type="internal", \
+ org.apache.taglibs.standard.tag.common.xml;  type="internal", \
+ org.apache.taglibs.standard.tag.el.core;  type="internal", \
+ org.apache.taglibs.standard.tag.el.fmt;  type="internal", \
+ org.apache.taglibs.standard.tag.el.sql;  type="internal", \
+ org.apache.taglibs.standard.tag.el.xml;  type="internal", \
+ org.apache.taglibs.standard.functions;  type="internal", \
+ org.apache.taglibs.standard.tag.rt.core;  type="internal", \
+ org.apache.taglibs.standard.tag.rt.fmt;  type="internal", \
+ org.apache.taglibs.standard.tag.rt.sql;  type="internal", \
+ org.apache.taglibs.standard.tag.rt.xml;  type="internal", \
+ org.apache.el;  type="internal", \
+ org.apache.el.lang;  type="internal", \
+ org.apache.jasper.el;  type="internal"
+IBM-ShortName: jsp-3.0
+IBM-SPI-Package: com.ibm.wsspi.jsp.taglib.config
+Subsystem-Name: Jakarta Server Pages 3.0
+-features=io.openliberty.jakarta.jsp-3.0, \
+ com.ibm.websphere.appserver.servlet-5.0; \
+ com.ibm.websphere.appserver.el-4.0
+-bundles=com.ibm.ws.org.eclipse.jdt.core.3.10.2.v20160712-0000, \
+ io.openliberty.jakarta.jstl.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.servlet:jstl:2.0", \
```

Comment:
maven co-ords look wrong, I think it should be `jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:2.0.0`